flake.nix

```diff
@@ -3,7 +3,7 @@
   description = "flake-iter";
 
   inputs = {
-    nixpkgs.url = "https://flakehub.com/f/NixOS/nixpkgs/0.2405.*";
+    nixpkgs.url = "https://flakehub.com/f/NixOS/nixpkgs/0.2411.*";
```

Comment:
note we can use `0` and it'll follow latest stable :)

Reply:
Ah, right, will change